src/DataTypes/Option/Option.cs

```diff
@@ -21,7 +21,7 @@ private Option(bool isSome, A value)
             _value = value;
         }
 
-        public static Option<A> None()
+        public static Option<A> None

```

Comment:
It's a little bit invasive this change.
The other concern is related to the behavior; I mean this statement is a factory method, so, to me, it's better to be a function.
What do you think?